tests/test_schema.py

```diff
@@ -217,6 +217,13 @@ def test_make_skeleton_schema(self):
         self.schema_obj.make_skeleton_schema()
         self.schema_obj.validate_schema(self.schema_obj.schema)
 
+    def test_make_skeleton_schema_absent_name(self):
+        """Test making a new schema skeleton"""
+        self.schema_obj.schema_filename = self.template_schema
+        self.schema_obj.pipeline_manifest["description"] = "Test pipeline"
+        self.schema_obj.make_skeleton_schema()
+        self.schema_obj.validate_schema(self.schema_obj.schema)
+
```

Comment:
I don't think this test actually checks that the `name` in the schema is what we expect it to be (which was part of the issue here). 

Probably need to add some sort of 
```python
assert self.schema_obj.schema['name'] == "expected name"
```

Reply:
Yes, I think the name is used for creating the `Title` attribute in the schema, there is no name attribute.